src/widgets/colorchooser.h

```diff
@@ -0,0 +1,251 @@
+#ifndef COLORCHOOSER_H
+#define COLORCHOOSER_H
+
+#include <QLineEdit>
+#include <QProxyStyle>
+#include <QWidget>
+#include <qapplication.h>
```

Comment:
Please use the correct form of capitalization, also, try to order includes in alphabetical order.